scirpt.js

```diff
@@ -9,4 +9,18 @@ humburger.addEventListener('click', () => {
 document.querySelectorAll('.nav-links').forEach((n) => n.addEventListener('click', () => {
   humburger.classList.remove('active');
   navMenu.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+// validate Email //
+
+const form = document.querySelector('.form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const emailel = document.getElementById('email').value;
+  const valid = document.querySelector('.invalid');
+  if (emailel !== emailel.toLowerCase()) {
+    valid.innerText = 'please enter a correct email address format';
+  } else {
+    form.submit();
```

Comment:
- [ ] Kindly fix this error:
![image](https://github.com/abdozayan12/portfolio-include-js/assets/24703171/98cfb526-c699-4034-bb37-67609b4ecde1)

this error happens when all input fields are valid and ready for submission because of this `form.submit();` line.

### Hint:
- The element with class `form` is a `div`, not a `form` so you can't use submit method with it.